docs/source/variables.ipynb

```diff
@@ -160,6 +160,79 @@
     "from featuretools.variable_types import list_variable_types\n",
     "list_variable_types()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Defining Custom Variable Types"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Users can define their own variables types. For example, to make a custom variable type called Age, do the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from featuretools.variable_types import Variable\n",
+    "\n",
+    "class Age(Variable):\n",
+    "    _default_pandas_dtype = float"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> The `_default_pandas_dtype` specifies the pandas dtype to use to represent the underlying data. You can find a list of pandas dtypes [here](https://pandas.pydata.org/pandas-docs/stable/getting_started/basics.html#dtypes)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Age** can now be used as a variable type to create a custom primitive. For example, let's create a transform primitive that returns a boolean if the age is greater than 100."
```

Comment:
maybe we could also mention it can be used when creating an entity

Reply:
Fixed:
https://github.com/FeatureLabs/featuretools/pull/1066/commits/601a412e6e8abe38cf357506fa5055bd3fe1a8c0#diff-788cca9c6a491b8eed11068ab47ed94cR201